woocommerce/qit-cli
# Allow to use extension slugs in the "--additional_woo_plugins" parameter

Make the behavior of `--additional_woo_plugins` consistent by also accepting a comma-separated list of slugs. Previously, it would only accept IDs.


### How to test this:

- See the unit tests that are in place, the snapshot tests represent the payload that is sent to the manager, see that it sends the extension IDs, even though we are passing in the slugs as input.

You can also test this by running a command such as this:

- `./qit run:e2e automatewoo --additional_woo_plugins=woocommerce-product-addons,woocommerce-min-max-quantities`
- And assert that it starts the test correctly, with the provided extensions
